holder.yml

```diff
@@ -56,6 +56,39 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /exchanges/:
+    get:
+      summary: provides a discovery endpoint for the exchanges supported by this sserver endpoint.
+      operationId: discoverExchanges
+      description: 
+        This endpoint returns a map of the exchange-ids (endpoints) supported by this server, 
+        and the associated protocol supported by that exchange endpoint in an 
+        "exchange-id" -> "protocol" map. The keys in the returned object MUST be a valid 
+        ExchangeId path parameter, and the value field MUST be a string that 
+        references the supported protocol on that endpoint. Protocol strings MUST be 
+        the protocol specification name or type, or a url to a spec. Clients consuming this 
+        list wishing to use an exchange endpoint MUST recognize and support the protocol identified
+        in the value field. Clients are not expected to dynamically process the protocol specified.    
+      responses:
+        "200":
+          description: a map of the exchange-id endpoints to protocols those exchanges support.
+          content:
+            application/json:
+              schema:
+                type: object 
+                additionalProperties: 
+                  type: string
+              example:
+                {
+                  "credential-refresh" : "CredentialRefresh2020",
```

Comment:
The way this is expressed in a VC is the following:
```
    "refreshService": {
      "type": "UnmediatedRefresh2021",
      "url": "https://example.edu/exchanges/refresh-degree",
      "validAfter": "2021-09-01T19:23:24Z"
    },
```

could map to this like so:

```
    "https://example.edu/exchanges/refresh-degree": {
      "type": "UnmediatedRefresh2021",
    },
```

Could be a proper JSON-LD object with semantics (not saying this is a good idea, just trying it out):

```
  "@context": ["https://w3id.org/vc/api/v1",  "https://w3id.org/vc-refresh-2021/v1"],
  "service": [{
    "type": "UnmediatedRefresh2021", 
    "serviceEndpoint": "https://example.edu/exchanges/refresh-degree"
  }]
```